benchmark/buffers/buffer-swap.js

```diff
@@ -3,37 +3,53 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  method: ['swap16', 'swap32', 'htons', 'htonl'],
-  len: [4, 64, 512, 768, 1024, 1536, 2056, 4096, 8192],
-  n: [1e6]
+  method: ['swap16', 'swap32', 'swap64'/*, 'htons', 'htonl', 'htonll'*/],
```

Comment:
Yep, see comment a few lines down. The commented-out ones are only for picking the crossover point between the JS and C++ implementations, and make the benchmarks super slow.